addons/pos_adyen/views/adyen_account_views.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="adyen_account_view_form" model="ir.ui.view">
+        <field name="name">adyen.account.view.form.inherit.pos.adyen</field>
+        <field name="model">adyen.account</field>
+        <field name="inherit_id" ref="adyen_marketpay.adyen_account_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//header" position="inside">
+                <button name="action_order_terminal" string="Order a Terminal" class="oe_highlight" type="object"/>
```

Comment:
it is visible even when the account is not created, shouldn't we show this button only when the account is validated by adyen, or at least when the account is created?